mcs/class/System/Test/System.Net.NetworkInformation/IPInterfacePropertiesTest.cs

```diff
@@ -0,0 +1,73 @@
+//
+// IPInterfacePropertiesTest.cs - NUnit Test Cases for System.Net.NetworkInformation.IPInterfaceProperties
+//
+// Authors:
+//   Ben Woods (woodsb02@gmail.com)
+//
+
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Net.NetworkInformation;
+
+namespace MonoTests.System.Net.NetworkInformation
+{
+
+[TestFixture]
+public class IPInterfacePropertiesTest
```

Comment:
Seems some of the very old test classes didn't indent this and the TemplateTest.cs was last touched more than 10 years ago. The newer classes [do indent](https://github.com/mono/mono/search?l=csharp&q=TestFixture&utf8=%E2%9C%93) however. I guess we need to fix the templates.